microhapdb/cli/marker.py

```diff
@@ -92,7 +92,6 @@ def display(result, view_format, delta=10, minlen=80, extend_mode=0, trunc=True,
 
 
 def subparser(subparsers):
-    desc = microhapdb.cli.bubbletext + "\nRetrieve marker records by identifier or query"
```

Comment:
Bubbletext did not render well on the sphinx autodocs. It's still shown when `microhapdb -h` is run, but no longer when e.g. `microhapdb marker -h` is run.